controllers/imageclusterinstall_controller_test.go

```diff
@@ -32,6 +32,21 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func getBHM(state bmh_v1alpha1.ProvisioningState) *bmh_v1alpha1.BareMetalHost {
```

Comment:
This should probably be `getBMH`, right?

Reply:
Maybe also just rename to `bmhInState` or something to avoid it sounding like this is querying for a BMH.